services/feed/feed.spec.js

```diff
@@ -14,6 +10,15 @@ describe("getdata FUNC TEST", () => {
     )
     expect(typeof result).toBe("object")
   })
+
+  it("it should return empty array", async () => {
+    const result = await service.getData(
```

Comment:
The unit test is calling the actual dependent function `readFile`. 
Please use mock to test only `getData`.

Also add the necessary unit tests to test `readFile` from the service